src/Nethermind/Nethermind.Consensus/Requests/DepositsProcessor.cs

```diff
@@ -0,0 +1,51 @@
+// SPDX-FileCopyrightText: 2022 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using System.Collections.Generic;
+using Nethermind.Abi;
+using Nethermind.Core;
+using Nethermind.Core.ConsensusRequests;
+using Nethermind.Core.Specs;
+using System.Linq;
+using Nethermind.Core.Extensions;
+
+namespace Nethermind.Consensus.Requests;
+
+public class DepositsProcessor : IDepositsProcessor
+{
+    private AbiSignature depositEventABI = new("DepositEvent", AbiType.DynamicBytes, AbiType.DynamicBytes, AbiType.DynamicBytes, AbiType.DynamicBytes, AbiType.DynamicBytes);
```

Comment:
use Abi.UInt64 for relevant fields!

Reply:
Every field is of `bytes` type: https://etherscan.io/address/0x00000000219ab540356cbb839cbe05303d7705fa#code